src/linear_algebra/finite_dimensional.lean

```diff
@@ -725,21 +727,21 @@ begin
 end
 
 /-- The sum of the dimensions of s + t and s ∩ t is the sum of the dimensions of s and t -/
-theorem dim_sup_add_dim_inf_eq (s t : submodule K V)
+theorem rank_sup_add_rank_inf_eq (s t : submodule K V)
   [finite_dimensional K s] [finite_dimensional K t] :
   finrank K ↥(s ⊔ t) + finrank K ↥(s ⊓ t) = finrank K ↥s + finrank K ↥t :=
 begin
   have key : module.rank K ↥(s ⊔ t) + module.rank K ↥(s ⊓ t) =
-    module.rank K s + module.rank K t := dim_sup_add_dim_inf_eq s t,
-  repeat { rw ←finrank_eq_dim at key },
+    module.rank K s + module.rank K t := rank_sup_add_rank_inf_eq s t,
+  repeat { rw ←finrank_eq_rank at key },
   norm_cast at key,
   exact key
 end
 
-lemma dim_add_le_dim_add_dim (s t : submodule K V)
+lemma rank_add_le_rank_add_rank (s t : submodule K V)
```

Comment:
This lemma and a few others still have the wrong name; but I intend to fix `rank`/`finrank` confusion in a follow-up. These misnomers will be more obvious after this PR is in.